examples/enclave_manager/setup.py

```diff
@@ -29,16 +29,13 @@
 tcf_root_dir = os.environ.get('TCF_HOME', '../../../../')
 enclave_dir = os.path.realpath(os.path.join(tcf_root_dir, 'tc/sgx/trusted_worker_manager/enclave'))
 
-module_path = 'tc/sgx/trusted_worker_manager/enclave_wrapper'
+module_path = 'tc/sgx/trusted_worker_manager/enclave_wrapper/avalon_enclave_sgx'
 module_src_path = os.path.join(tcf_root_dir, module_path)
 
-sgx_sdk_env = os.environ.get('SGX_SDK', '/opt/intel/sgxsdk')
-sgx_ssl_env = os.environ.get('SGX_SSL', '/opt/intel/sgxssl')
-sgx_mode_env = os.environ.get('SGX_MODE', 'SIM')
-if not sgx_mode_env or (sgx_mode_env != "SIM" and sgx_mode_env != "HW"):
-    print("error: SGX_MODE value must be HW or SIM, current value is: ", sgx_mode_env)
-    sys.exit(2)
+enclave_bridge_path = os.path.join(tcf_root_dir,
+        'tc/sgx/trusted_worker_manager/enclave_wrapper/avalon_enclave_bridge')
 
+sgx_ssl_env = os.environ.get('SGX_SSL', '/opt/intel/sgxssl')
```

Comment:
We should avoid SGX SSL dependency in python.

Reply:
Done